ddtrace/debugging/_exception/auto_instrument.py

```diff
@@ -210,3 +212,16 @@ def on_span_finish(self, span: Span) -> None:
 
             span.set_tag_str(DEBUG_INFO_TAG, "true")
             span.set_tag_str(EXCEPTION_ID_TAG, str(exc_id))
+
+
+def capture_exception() -> None:
+    import sys
+
+    from ddtrace import tracer
+
+    try:
+        span = tracer.current_span()
```

Comment:
if trace._current_span() is not the service entry, will this still work? 

I heard there is a tag ET support to index that span error if it not the service entry. maybe this should also set that tag. However, I can't seem to find it on their docs. so maybe this should add tags to service entry span (or both entry and current) or simple abort for non-entry?

Reply:
> if trace._current_span() is not the service entry, will this still work?

This will put debug info on the current span, regardless of what the span is. So any limitations are probably not due to "us" I'd expect

Reply:
yup. this is broken as non entry spans are not get indexed and there is no way for the user to query on that and it would not appear in Error tracking. so not sure why we allow the user to do it in the first place.